modules/docs/mdx/CONTRIBUTING.mdx

```diff
@@ -54,38 +49,25 @@ If you are contributing code, please take a look at the following sections to fa
 with how the Canvas Kit repo is organized and run. This will help streamline the pull request
 process. If you have any questions, please reach out!
 
-If your contribution is visual or UI-based, please ensure you consult with a designer so the
-contribution can be evaluated as an end-to-end process.
-
-#### Automation on Commit
```

Comment:
The next few sections were moved under a new section called `Developer Experience`.